gmsh_interop/runner.py

```diff
@@ -149,12 +159,18 @@ def get_gmsh_version_from_string(output: str) -> Version | None:
 
 
 class GmshRunner:
-    def __init__(self, source, dimensions=None, order=None,
-            incomplete_elements=None, other_options=(),
-            extension="geo", gmsh_executable="gmsh",
-            output_file_name=None,
-            target_unit=None,
-            save_tmp_files_in=None):
+    def __init__(
+            self,
+            source: str | ScriptSource | FileSource | ScriptWithFilesSource,
+            dimensions: Literal[1, 2, 3] | None = None,
```

Comment:
I'm not quite sure using a `Literal[1, 2, 3]` here is a good idea?

Reply:
I think I'd prefer `int`. I'll plug it in.

Reply:
Done.

Reply:
Thanks! :grin: